Vostok.ServiceDiscovery.Telemetry.Hercules/TagNames.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Vostok.ServiceDiscovery.Telemetry.Hercules
+{
+    internal static class TagNames
+    {
+        public const string Application = "application";
+        public const string Environment = "environment";
+        public const string Replica = "replica";
+        public const string Kind = "serviceDiscoveryEventKind";
```

Comment:
```suggestion
        public const string Kind = "kind";
```

не помню, но вроде уже оставлял такое